python/kfserving/docs/KFServingClient.md

```diff
@@ -277,12 +278,53 @@ timeout_seconds | int | Timeout seconds for watching. Defaults to 600. | Optiona
 ### Return type
 object
 
+## rollout_canary
+> rollout_canary(name, percent=100, canary=None, namespace=None, watch=False, timeout_seconds=600)
+
+Rollout canary for the created KFService in the specified namespace. The `rollout_canary` API updates the KFService `canaryTrafficPercent` to speficed `percent` to adjust the traffic percent that will be distributed on the canary version. If the argument `percent` is not speficed, the API will set `canaryTrafficPercent` to 100. The `rollout_canary` also supports setting or updating `canary` endpoint spec for the KFServing. If the KFService has no `canary` endpoint spec, need to specify with the API it while setting `canaryTrafficPercent`.
+
+The difference between [rollout_canary](#rollout_canary) and [promote](#promote) is that the `rollout_canary` only updates `canaryTrafficPercent` (or  `canary` endpoint spec) while the `promote` moves the `canary` version of the KFService to `default`.
+
+
+### Example
+
+```python
+
+KFServing = KFServingClient()
+KFServing.rollout_canary('flower-sample', percent=50, namespace='kubeflow')
+
+# The API also supports setting or updating the canary endpoint spec.
+canary_spec = V1alpha2EndpointSpec(
+  predictor=V1alpha2PredictorSpec(
+    tensorflow=V1alpha2TensorflowSpec(
+    storage_uri='gs://kfserving-samples/models/tensorflow/flowers')))
+
+KFServing.rollout_canary('flower-sample', percent=50, canary=canary_spec, namespace='kubeflow')
+
+# The API also supports watching the promoted KFService status till it's READY.
+# KFServing.rollout_canary('flower-sample', percent=50, namespace='kubeflow', watch=True)
+```
+
+### Parameters
+Name | Type |  Description | Notes
+------------ | ------------- | ------------- | -------------
+name  | str | The KFService name for promoting.| |
+percent  | int | The traffic percent that will be distributed on the canary version.| |
+canary  | [V1alpha2EndpointSpec](V1alpha2EndpointSpec.md) | The canary endpoint spec for KFServing.| |
+namespace | str | The KFService's namespace. If the `namespace` is not defined, will align with KFService definition, or use current or default namespace if namespace is not specified in KFService definition. | Optional|
+watch | bool | Watch the promoted KFService if `True`, otherwise will return the promoted KFService object. Stop watching if KFService reaches the optional specified `timeout_seconds` or once the KFService overall status `READY` is `True`. | Optional |
```

Comment:
suggest dropping word `promoted`, it is not promoted yet right?

Reply:
Yes, my mistake.